golem-api-grpc/src/lib.rs

```diff
@@ -218,23 +218,9 @@ pub mod proto {
 
     #[cfg(test)]
     mod tests {
-        use test_r::test;
-
         use crate::proto::golem;
         use prost::Message;
-        use std::str::FromStr;
-
-        #[test]
-        fn test_uuid() {
```

Comment:
No assertion here

Reply:
Should we put assertions or delete this? :)